src/main/resources/db/migration/V013__Add_new_fields.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE envelopes
+  ADD COLUMN caseNumber VARCHAR(100) NULL;
+
+ALTER TABLE non_scannable_items
+  ADD COLUMN documentControlNumber VARCHAR(100) NOT NULL;
+
+ALTER TABLE payments
+  ADD COLUMN paymentInstrumentNumber VARCHAR(100),
+  ADD COLUMN sortCode VARCHAR(10) NULL DEFAULT 112233,
+  ADD COLUMN accountNumber VARCHAR(50) NULL DEFAULT 12345678;
```

Comment:
why the defaults if column in nullable?

Reply:
I added it for "Cheque" payment methods.  but I removed Default values and added an update statement